torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -257,12 +266,14 @@ def _pre_forward(
     handles: List[FlatParamHandle],
     unshard_fn: Callable,
     module: nn.Module,
-    input: Any,
-):
+    args: Tuple[Any],
```

Comment:
nit: Does this need to be `Tuple[Any, ...]`? (also affects docstring)

My understanding is that `Tuple[Any]` means a singleton tuple with an element of any type and that `Tuple[Any, ...]` means a tuple with arbitrarily many elements, where each element may be any type.

Reply:
Oh Yep, thanks for the catch!